README.rst

```diff
@@ -36,43 +36,6 @@ From conda-forge::
 
     $ conda install -c conda-forge pynamodb
 
-Upgrading
```

Comment:
Moving this to docs since it's not relevant to most users. I'm keeping a warning with a link on the README